ani-cli

```diff
@@ -2,6 +2,26 @@
 
 version_number="4.6.1"
 
+# directory for anime episode timestamp files
+
+# directory to store files named as anime titles with episodes mapped with time left on it
+ep_timestamp_dir="${HOME}/.local/state/ani-cli/ani-eps-hsts/"
+
+# mpv lua script for ani-cli that keeps track of episodes
+ani_cli_scripts_dir="${HOME}/.config/ani-cli/scripts/"
+for dir in $ep_timestamp_dir $ani_cli_scripts_dir
+do
+    [[ ! -e "$dir" ]] && mkdir -p "$dir"
+    if [[ ! -d "$dir" ]]
+    then
+        rm -rf "$dir"
+        mkdir -p "$dir"
+    fi
+done
+unset ep_timestamp_dir ani_cli_scripts_dir
+
+
+
```

Comment:
Setting timestamp directory and creating it if it not exists - this should be all done in the helper script. It doesn't belong here. Also the use of XDG environment variables is encouraged (like with history).

(bonus: shellcheck will definitely complain over ``[[``, as it's a non-posix feature)